src/object.cr

```diff
@@ -201,6 +201,23 @@ class Object
     yield self
   end
 
+  # Returns `true` if `self` is included in the *collection* argument.
+  #
+  # ```
+  # 10.in?(0..100)     # => true
+  # 10.in?({0, 1, 10}) # => true
+  # 10.in?(0, 1, 10)   # => true
+  # 10.in?(:foo, :bar) # => false
+  # ```
+  def in?(collection : Enumerable) : Bool
```

Comment:
This means `Object` depends on `Enumerable`, and `vice-versa`.
Furthermore, any object that implements `includes?` is valid, not only enumerables.
Better to drop this type annotation.